redis-hbo-provider/pom.xml

```diff
@@ -289,15 +289,19 @@
             <plugin>
                 <groupId>org.basepom.maven</groupId>
                 <artifactId>duplicate-finder-maven-plugin</artifactId>
+                <version>1.2.1</version>
                 <configuration>
                     <ignoredResourcePatterns>
+                        <ignoredResourcePattern>META-INF.versions.9.module-info</ignoredResourcePattern>
+                        <ignoredResourcePattern>module-info</ignoredResourcePattern>
                         <ignoredResourcePattern>com.google.*$</ignoredResourcePattern>
                         <ignoredResourcePattern>.*\.proto$</ignoredResourcePattern>
                         <ignoredResourcePattern>about.html</ignoredResourcePattern>
                         <ignoredResourcePattern>mozilla/public-suffix-list.txt</ignoredResourcePattern>
                     </ignoredResourcePatterns>
                     <ignoredClassPatterns>
                         <ignoredClassPattern>module-info</ignoredClassPattern>
+                        <ignoredClassPattern>META-INF.versions.9.module-info</ignoredClassPattern>
```

Comment:
Can you make `<ignoredClassPatterns combine.children="append">` so we don't duplicate this in the submodules?

Reply:
done